src/test/java/de/jetwick/snacktory/ArticleTextExtractorTest.java

```diff
@@ -47,7 +47,7 @@ public void testData1() throws Exception {
     public void testData2() throws Exception {
         // http://benjaminste.in/post/1223476561/hey-guys-whatcha-doing
         JResult res = extractor.extractContent(readFileAsString("test_data/2.html"));
-        assertEquals("BenjaminSte.in - Hey guys, whatcha doing?", res.getTitle());
+        assertEquals("BenjaminSte.in", res.getTitle());
```

Comment:
Let's get rid of this unit test. The website is not online anymore and the HTML structure is probably obsolete too.